packages/@azure/servicebus/data-plane/test/streamingReceiver.spec.ts

```diff
@@ -50,6 +50,24 @@ function unExpectedErrorHandler(err: Error): void {
   }
 }
 
+// Maximum wait duration for the Streaming Receiver to receive the messages = `10000 ms`(10 seconds)
+const maxDelayForStreamingReceiver = 10000;
+// Keep checking whether the boolCheck is true after every `1000 ms`(1 second)
+const delayBetweenRetriesForStreamingReceiver = 1000;
+
+async function DelayStreaming(
+  boolCheck: () => boolean,
+  delayBetweenRetries: number,
+  maxWaitTime: number
+): Promise<boolean> {
+  const maxTime = new Date().getTime() + maxWaitTime;
+  while (new Date().getTime() < maxTime) {
```

Comment:
`Date.now()` works too?

Reply:
Agreed, let use `Date.now()` instead of `new Date().getTime()`